concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -107,8 +107,8 @@ resources:
   - name: graphite-nozzle
     type: git
     source:
-      uri: https://github.com/alphagov/graphite-nozzle
-      branch: 106191208-deaggregated_names
+      uri: https://github.com/alphagov/paas-graphite-nozzle
+      branch: upgrade_go
```

Comment:
shall we have some sort of "gds-master" that we merge against? Are we even interested to push changes upstream? What happened to this nozzle after CC got merged into pivotal? Is it officially supported? Or scrapped?